### ISSUE TITLE
EMPTY_OPTS warning on require

### ISSUE THREAD

**Issue by soumyaray** [dry-rb/dry-validation#278]
As soon as I require dry-validation (0.10.3) in my code, or in pry, I get this warning:
```
/Users/soumyaray/.rvm/gems/ruby-2.3.1/gems/dry-validation-0.10.3/lib/dry/validation/message_compiler.rb:13: warning: already initialized constant Dry::Validation::MessageCompiler::EMPTY_OPTS
/Users/soumyaray/.rvm/gems/ruby-2.3.1/gems/dry-core-0.2.1/lib/dry/core/constants.rb:52: warning: previous definition of EMPTY_OPTS was here
```

**Comment by Kukunin** [comment]
duplication of https://github.com/dry-rb/dry-validation/issues/276

**Comment by solnic** [comment]
This is fixed in 0.10.4
